gentest/fixtures/YGMarginTest.html

```diff
@@ -39,3 +39,54 @@
   <div style="margin-bottom; 10px; flex-grow: 1;"></div>
   <div style="flex-grow: 1;"></div>
 </div>
+
+<div id="margin_auto_bottom" style="width: 200px; height: 200px; align-items: center;">
+  <div style="width: 50px; height: 50px;margin-bottom:auto"></div>
+  <div style="width: 50px; height: 50px;"></div>
+</div>
+
+<div id="margin_auto_top" style="width: 200px; height: 200px; align-items: center;">
+  <div style="width: 50px; height: 50px;margin-top:auto"></div>
+  <div style="width: 50px; height: 50px;"></div>
+</div>
+
+<div id="margin_auto_bottom_and_top" style="width: 200px; height: 200px; align-items: center;">
+  <div style="width: 50px; height: 50px;margin-top:auto;margin-bottom:auto"></div>
+  <div style="width: 50px; height: 50px;"></div>
+</div>
+
+<div id="margin_auto_bottom_and_top_justify_center" style="width: 200px; height: 200px; justify-content: center;">
+  <div style="width: 50px; height: 50px;margin-top:auto;margin-bottom:auto"></div>
+  <div style="width: 50px; height: 50px;"></div>
+</div>
+
+<div id="margin_auto_top_double" style="width: 200px; height: 200px; align-items: center;">
```

Comment:
add a similar test for row direction. Also please change name to something like `margin_auto_multiple_children` for a `_row` and `_column` suffix.